packages/xo-web/src/xo-app/home/index.js

```diff
@@ -456,6 +456,7 @@ const NoObjects = props =>
     mapValues(items, item => ({
       ...item,
       container: containers[item.$container || item.$pool],
+      usage: item.size > 0 ? (item.physical_usage / item.size) * 100 : item.physical_usage,
```

Comment:
IMHO, it does not makes sense to compare some SR by percentage and other by usage.

Reply:
What can we do if "size" is 0 or negative? 
and the progress bar is displayed only if the size is greater than 0.

Reply:
Good question, we have to make a choice there, either consider them as full or empty.